src/ngraph/runtime/intelgpu/intelgpu_backend.cpp

```diff
@@ -1629,11 +1629,11 @@ shared_ptr<runtime::Executable>
             const shared_ptr<op::ConvolutionBackpropFilters> conv_op =
                 static_pointer_cast<op::ConvolutionBackpropFilters>(op);
 
-            const Strides& win_stride = conv_op->get_window_movement_strides_backward();
-            const CoordinateDiff& pad_below = conv_op->get_padding_below_backward();
-            const CoordinateDiff& pad_above = conv_op->get_padding_above_backward();
-            const Strides& win_dilation = conv_op->get_window_dilation_strides_backward();
-            const Strides& data_dilation = conv_op->get_data_dilation_strides_backward();
+            const Strides& win_stride = conv_op->get_window_dilation_strides_forward();
```

Comment:
Logically, if you operate with "backprop" you expect something like "_backwards". If all these "backward" and "forward" has no meaning, please make the name shorter, without suffixes.

Reply:
That's the second cleanup I alliuded too; "window movement" versus "filter" etc.